### PR TITLE
refactor: make fromFile chainable

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,11 +36,15 @@
   },
   "devDependencies": {
     "@zazuko/rdf-vocabularies": "^2019.10.22",
+    "@tpluscode/rdf-string": "^0.2.7",
+    "@types/sinon": "^7.5.2",
```
Comment (bergos) [inline package.json]: Are these deps required?

Comment (tpluscode) [reply package.json]: Ah, yes, `rdf-string` is not used. Must have installed to inspect something but shouldn't have added to package.json

The other one? Well, I don't have the entire Sinon memorized. `@types` greatly improves the IDE suggestions in webstorm

Comment (tpluscode) [reply package.json]: @bergos

**Comment by tpluscode** [comment]
replaced by #62

**Pull request closed without merging**